dev-lang/fpc/fpc_bin-3.0.2.recipe

```diff
@@ -0,0 +1,111 @@
+SUMMARY="Pre-built binaries for the FPC build"
```

Comment:
Should be "Professional Pascal compiler"

Reply:
OK